scripts/ingests/sanghi23/sanghi23_modeled_parameters_skipped.csv

```diff
@@ -0,0 +1,16 @@
+# %ECSV 1.0
+# ---
+# datatype:
+# - {name: Skipped, datatype: string}
+# - {name: Reason, datatype: string}
+# delimiter: ','
+# schema: astropy-2.0
+Skipped,Reason
+veryred-1527,"no source found, atmo model"
```

Comment:
Is there really a source name called "veryred-1527"?  @cosmicoder, any insight on what source this is?